### PR TITLE
Add new keyboard "Radialex"

### PR DESCRIPTION
**Pull request by takashicompany**
## Description
Radialex is 40% larger keyboard.  
The radial key layout allows you to place your hand in a natural position and move your fingers the shortest distance to type.  
Your iconic keys and keycaps can be placed in the center of the keyboard.  
It is possible to make it shine with LED backlighting.  
The keyswitches can also be replaced with MX sockets.  

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by takashicompany** [comment]
@drashna  
What is the status of this Pull Request?

**Comment by drashna** [comment]
> What is the status of this Pull Request?

Mostly, it's waiting on a second review.

File: keyboards/takashicompany/radialex/rules.mk
Comment (fauxpark) [inline keyboards/takashicompany/radialex/rules.mk:18]:
```suggestion
NKRO_ENABLE = no            # Enable N-Key Rollover
```

File: keyboards/takashicompany/radialex/keymaps/via/keymap.c
Comment (fauxpark) [inline keyboards/takashicompany/radialex/keymaps/via/keymap.c:86]:
```suggestion
            return false;
```

File: keyboards/takashicompany/radialex/keymaps/default/keymap.c
Comment (fauxpark) [inline keyboards/takashicompany/radialex/keymaps/default/keymap.c:58]:
```suggestion
                rgblight_toggle();
```

File: keyboards/takashicompany/radialex/keymaps/default/keymap.c
Comment (fauxpark) [inline keyboards/takashicompany/radialex/keymaps/default/keymap.c:60]:
```suggestion
            return false;
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/takashicompany/radialex/keymaps/default/keymap.c
Comment (takashicompany) [reply keyboards/takashicompany/radialex/keymaps/default/keymap.c]: Fixed! https://github.com/qmk/qmk_firmware/pull/14949/commits/cbf3710a0e1b180dfd12c588a7b376c7c1d612c7

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**
